x-pack/plugins/canvas/public/components/page_manager/page_manager.js

```diff
@@ -158,16 +158,18 @@ export class PageManager extends React.PureComponent {
                 >
                   {Style.it(
                     workpadCSS,
-                    <PagePreview
-                      isWriteable={isWriteable}
-                      page={page}
-                      height={100}
-                      pageNumber={pageNumber}
-                      movePage={movePage}
-                      selectedPage={selectedPage}
-                      duplicatePage={duplicatePage}
-                      confirmDelete={this.confirmDelete}
-                    />
+                    <div>
```

Comment:
I removed another div higher up, but I realized that I forgot to add one here. `style-it` expects a wrapper `<div>` around your element you're trying to style, so it can inject an id that it uses to apply the styles. Without the div, the styles weren't being applied to the thumbnails at all, which was an unreported bug, it turns out.

Reply:
Funny, just adding that div fixes the issue too. Still good to default the value though.